SEED-platform/seed
# Fix mapping previews for files with a single column

#### Any background context you want to provide?
We've had a bug _for a long time_ that prevented seeing the first five rows during mapping of a file with a single column of data

#### What's this PR do?
Fixes the five-row preview when mapping a file with a single column of data

#### How should this be manually tested?
1. Create a file with a single column of data, e.g.:
    ```csv
    Custom ID 1
    120243 E True Lane
    95373 E Peach Avenue
    95864 SW Cottonwood Court
    155371 W Cherry Loop
    180821 S Tyler Lane
    ```
2. Upload the file and go to the mapping page
3. The first-five-rows preview should show the full values, not just the first letter of each

#### Screenshots (if appropriate)

Before and after:

![Single-Column-Import](https://github.com/SEED-platform/seed/assets/411466/88ad8c7c-d26c-4a2d-b813-3aecdb7ef8e2)
